addons/message_update_v2.22/manifest.json

```diff
@@ -11,15 +11,38 @@
   },
   "message": {
     "date": 1713466835,
-    "id": "message_update_v2.22",
+    "id": "message_update_v2.22-0",
```

Comment:
Note that the message ID must change in order for the new subtitle string to register in pontoon.